### PR TITLE
CIV-14010 Service PR for Email notification: Defendant LR Claim Paid in Full

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14010

### Change description ###

Email notification: Defendant LR Claim Paid in Full

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by German-A-Amoros — APPROVED** [review]

File: build.gradle
Comment (hmcts-version1-ignacio) [inline build.gradle:1]: Remember not to merge this.

**Review by hmcts-version1-ignacio — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java [hunk 1]
```diff
@@ -949,9 +949,7 @@ public class FlowStateAllowedEventService {
                 asyncStitchingComplete,
                 TRANSFER_ONLINE_CASE,
                 INVALID_HWF_REFERENCE,
-                COURT_OFFICER_ORDER,
-                SETTLE_CLAIM,
-                SETTLE_CLAIM_MARK_PAID_FULL
```
[the remark this answers is not in the document]
Comment (Omaira-Melo-Hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java:954]: My PR build is fixed

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -337,7 +337,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.97'
+  implementation 'com.github.hmcts:civil-commons:v1.0.91_CIV_14010_4'
```
Comment (sjamihmcts) [inline build.gradle:340]: To be updated once civil-commons PR is merged

**Review by ShwetaTandel-hmcts — DISMISSED** [review]

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

File: build.gradle
Comment (sjamihmcts) [reply build.gradle:1]: Updated the version with latest civil-commons that is released.

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**